### PR TITLE
Add myDashboard  functional

### PR DESCRIPTION
**Pull request by DimaUstim**

### CODE REVIEW

File: src/app/components/my-dashboard/mydashboard.component.ts [hunk 1]
```diff
@@ -1,15 +1,85 @@
...
   selector: 'app-mydashboard',
   templateUrl: './mydashboard.component.html',
-  styleUrls: ['./mydashboard.component.scss']
+  styleUrls: ['./mydashboard.component.scss'],
+  providers: [MessageService]
 })
 export class MydashboardComponent implements OnInit {
+  pets: Pet[] = [];
+  loading: boolean = true;
+  activityValues: number[] = [0, 100];
```
Comment (AndriiPanasenkoSolvve) [inline src/app/components/my-dashboard/mydashboard.component.ts]: Where is it used?

File: src/app/components/my-dashboard/mydashboard.component.ts [hunk 2]
```diff
@@ -1,15 +1,85 @@
...
+  globalFilter(event: any, dataTable: any) {
+    dataTable.filterGlobal(event.target?.value, 'contains');
+  }
+
+  openContextMenu(event: any, cm: any, pet: any) {
+    event.preventDefault();
+    event.stopPropagation();
+    cm.show(event);
+    this.selectedPet = pet;
+    return false;
```
Comment (AndriiPanasenkoSolvve) [inline src/app/components/my-dashboard/mydashboard.component.ts]: What does this result mean?

File: src/app/components/my-dashboard/mydashboard.component.ts [hunk 3]
```diff
@@ -1,15 +1,85 @@
...
+        },
+      }
+    ];
   }
 
+  async deletePet(selectedPet: any) {
+    this.pets = this.pets.filter((p) => p.id !== selectedPet.id);
+    await this.petsService.deletePet(selectedPet.id);
+    this.messageService.add({ severity: 'info', summary: 'Pet Deleted', detail: selectedPet.name });
+    this.showList();
```
Comment (AndriiPanasenkoSolvve) [inline src/app/components/my-dashboard/mydashboard.component.ts]: There is no need to load pets again after one of them has been deleted. You've already updated the pets list (first line of function)

**Review by AndriiPanasenkoSolvve — CHANGES_REQUESTED** [review]
Please fix the minor issues

File: src/app/components/my-dashboard/mydashboard.component.ts (hunk 1 above)
Comment (DimaUstim) [reply src/app/components/my-dashboard/mydashboard.component.ts]: Fixed

File: src/app/components/my-dashboard/mydashboard.component.ts (hunk 2 above)
Comment (DimaUstim) [reply src/app/components/my-dashboard/mydashboard.component.ts]: Fixed

File: src/app/components/my-dashboard/mydashboard.component.ts (hunk 3 above)
Comment (DimaUstim) [reply src/app/components/my-dashboard/mydashboard.component.ts]: Fixed

**Review by AndriiPanasenkoSolvve — APPROVED** [review]

**Review by Maksym-Antsybor — APPROVED** [review]

**Pull request merged**